students/TimPauley/session03/MailRoomPart1.py

```diff
@@ -0,0 +1,111 @@
+#Tim Pauley
```

Comment:
not bothering to look at this -- I'll just review Part2
(but still give you credit)